src/schematools/contrib/django/factories.py

```diff
@@ -196,7 +192,26 @@ def field_kwargs(self):
             target_field = self.field.related_table.get_field_by_id(target_field_ids[0])
 
             if target_field_ids[0] != "id" and not target_field.is_primary:
-                kwargs["to_field"] = to_snake_case(target_field_ids[0])
+                return target_field_ids[0]
+
+        return None
+
+    @property
+    def field_kwargs(self):
+        # In schema foreign keys should be specified without _id,
+        # but the db_column should be with _id
+        kwargs = {
+            **super().field_kwargs,
+            "db_column": f"{to_snake_case(self.field.name)}_id",
```

Comment:
```suggestion
            "db_column": to_snake_case(self.field.name) + "_id",
```

Easier to read, IMHO.

Reply:
good point. This code wasn't changed, will check later (more PRs coming)